web_s2i/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:18.04
```

Comment:
any reason not to use the public python image?

Reply:
nop